library/ssl_tls12_client.c

```diff
@@ -3235,6 +3389,42 @@ static int ssl_write_client_key_exchange( mbedtls_ssl_context *ssl )
     {
         header_len = 4;
 
+#if defined(MBEDTLS_USE_PSA_CRYPTO)
+        unsigned char *out_p = ssl->out_msg + header_len;
+        unsigned char *end_p = ssl->out_msg + MBEDTLS_SSL_OUT_CONTENT_LEN -
+                               header_len;
+        psa_status_t status;
+        size_t output_offset = 0;
+        size_t output_len;
+
+        for( psa_pake_step_t step = PSA_PAKE_STEP_KEY_SHARE ;
+             step <= PSA_PAKE_STEP_ZK_PROOF ;
+             ++step )
+        {
+            /* For each step, prepend 1 byte with the length of the data */
+            if (step != PSA_PAKE_STEP_ZK_PROOF) {
+                *(out_p + output_offset) = 65;
```

Comment:
Is this `PSA_TLS12_ECJPAKE_TO_PMS_INPUT_SIZE`?

Reply:
Albeit size/value matches, I'm not sure that the define you mentioned is the right value here.

I took that value this [define](https://github.com/Mbed-TLS/mbedtls/blob/1b9cb62702a2b7293fcfcee80f92f21583601505/include/psa/crypto_extra.h#L1763) and even in this case raw values are used. This was introduced in PR #6390 which is quite recent.

Reply:
In case my comment above is correct and those defines are missing, what do you think if I add new ones?
Since those values are used in more than one place, it should make sense in my opinion

Reply:
Any of these options sound good.